odbc/src/api/util.rs

```diff
@@ -78,3 +79,20 @@ pub(crate) fn statement_attribute_to_string(attr: StatementAttribute) -> String
         StatementAttribute::SQL_ATTR_METADATA_ID => "METADATA_ID".to_string(),
     }
 }
+
+/// handle_sql_type returns the proper sql type for the application's current behavior. When the
+/// driver's version is set to ODBC 2.x, the datetime types must be mapped to different SQL types
+pub fn handle_sql_type(odbc_version: OdbcVersion, sql_type: SqlDataType) -> SqlDataType {
+    match odbc_version {
+        OdbcVersion::Odbc3 | OdbcVersion::Odbc3_80 => sql_type,
```

Comment:
It'd use _ here. Maybe one day, ODBC 4.0 makes it. We'd be covered in this case (unless of course they do yet another funky remapping of types)